### ISSUE TITLE
In-context forecasting results on 'M3 Others' are not as expected

### ISSUE THREAD

**Issue by hwanyu112** [thuml/AutoTimes#2]
在使用您提供的`scripts/in_context_forecasting/M3.sh`进行复现发现在M3 Year, Quarter, Month上的效果与paper中的基本一致，但是在M3 Others上得到的结果分别为：
zero shot： mape：9.140577      smape：8.3089
in context learning： mape：11.283587      smape：9.9321
使用time series prompt的in-context forecasting结果反而更差了，与paper中的结果相反。
请问造成这个结果的原因是什么呢？如果我有遗漏关键步骤麻烦请指出，谢谢！

**Comment by hvlgo** [comment]
Thank you for your feedback. May I ask on what type of GPU you are running?

**Comment by hwanyu112** [comment]
NVIDIA GeForce 24G RTX 4090

**Comment by hvlgo** [comment]
And what version of the transformers library are you using?

**Comment by hwanyu112** [comment]
4.31.0

**Comment by hvlgo** [comment]
It's quite strange; I tested it under the same conditions, and the results were consistent with those in the paper. `:thinking:`

**Comment by hvlgo** [comment]
Are the versions of other libraries consistent with those in `./requirements.txt`? I guess it might also be due to mixed precision.

**Comment by hvlgo** [comment]
Have you added the code mentioned in  [#1](https://github.com/thuml/AutoTimes/issues/1) ?

**Comment by hwanyu112** [comment]
Yes, and I have just changed my version of transformers to 4.35.2 in `./requirements.txt`. However, the results remained the same.

**Comment by hwanyu112** [comment]
Could you provide your checkpoint through Google Drive?

**Comment by hvlgo** [comment]
Sure, please wait for a short while.

**Comment by hwanyu112** [comment]
Using torch==2.0.1 addresses my problem. Thank you for your help!
